src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -789,6 +799,10 @@ private String getNumberOfWinnersRaw() {
     return rawConfig.rules.numberOfWinners;
   }
 
+  private String getStopTabulationEarlyOnRoundRaw() {
```

Comment:
If I'm not mistaken, this setting means that we'd stop _after_ the specified number of rounds, right? If so, let's call it that instead to make it more clear. Stop _on_ a round makes it sound like maybe we're stopping at the beginning of that round.